prisma/schema.prisma

```diff
@@ -157,6 +158,18 @@ model Wallet {
   @@index([userId])
 }
 
+model WalletLog {
+  id        Int @id @default(autoincrement())
+  createdAt DateTime @default(now()) @map("created_at")
+  userId    Int
+  user      User   @relation(fields: [userId], references: [id], onDelete: Cascade)
+  wallet    String
+  level     LogLevel
+  message   String
+
+  @@index([userId, createdAt])
+}
+
```

Comment:
I had to remove a foreign key to the wallet table since on errors, we don't create wallets.

Instead of doing some mental gymnastics to have a wallet that doesn't work in the wallet table, I simply abandoned the idea to have a link between the wallet table and the logs.

This also makes it possible to persist logs about wallets that have been detached.